scripts/genappimage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
```

Comment:
You are using `sh scripts/genappimage.sh` in Makefile. This can’t be `#!/bin/bash` in this case, sh is not bash in Ubuntu, it is dash.